base/intset.jl

```diff
@@ -239,3 +245,8 @@ function hash(s::IntSet, h::UInt)
     end
     h
 end
+
+minimum(s::IntSet) = first(s)
+maximum(s::IntSet) = last(s)
+extrema(s::IntSet) = (first(s), last(s))
+issorted(s::IntSet) = true
```

Comment:
Yes but is the order that Sets and IntSets are iterated guaranteed to stay fixed? Otherwise being sorted is not a property of the object. For the other structures you mentioned, there is clearly a well defined iteration order so `issorted` makes sense.